bin/activate.bat

```diff
@@ -1,78 +1,44 @@
-@echo off
-REM Check for CONDA_EVS_PATH environment variable
-REM It it doesn't exist, look inside the Anaconda install tree
-if "%CONDA_ENVS_PATH%" == "" (
-set CONDA_ENVS_PATH="%~dp0..\envs"
-)
+@echo on
+setlocal
 
-set CONDA_NEW_NAME=%~1
+set CONDA_NEW_ENV=%~1
 
-if "%~2" == "" goto skiptoomanyargs
-    echo ERROR: Too many arguments provided
-    goto usage
+if "%2" == "" goto skiptoomanyargs
+    (echo Error: did not expect more than one argument.) 1>&2
+    exit /b 1
 :skiptoomanyargs
 
-if "%CONDA_NEW_NAME%" == "" set CONDA_NEW_NAME=%~dp0..\
+if not "%1" == "" goto skipmissingarg
+    :: Set env to root if no arg provided
+    set CONDA_NEW_ENV=root
+:skipmissingarg
 
-REM Search through paths in CONDA_ENVS_PATH
-REM First match will be the one used
+REM Use conda itself to figure things out
 
-for %%F in ("%CONDA_ENVS_PATH:;=" "%") do (
-    if exist "%%~F\%CONDA_NEW_NAME%\conda-meta" (
-       set CONDA_NEW_PATH=%%~F\%CONDA_NEW_NAME%
-       goto found_env
-    )
-)
+SET "CONDA_EXE=%~dp0\conda.bat"
```

Comment:
Are you also using the development version of conda from conda/conda#1727 ? Note that you may need to manually delete/move/rename/whatever the <prefix>/Lib/site-packages/conda folder (and the conda-env folder there, too), or else the conda-installed packages take preference.  This has bitten me quite hard in debugging these packages.